### ISSUE TITLE
Wrap all thingie names in <em> tags in the "URL alias patterns" page.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#2333]
I accidentally found that we only do this for vocabulary names when I `dpm`ed in `/admin/config/search/path/patterns`:

![backdrop-only_vocabulary_names_wrapped_in_em_tags-dpm](https://cloud.githubusercontent.com/assets/2423362/20228818/3d836be6-a8a7-11e6-9e41-fca3ade3c643.png)

![backdrop-only_vocabulary_names_wrapped_in_em_tags](https://cloud.githubusercontent.com/assets/2423362/20228826/46bd0de8-a8a7-11e6-8403-20788e0b6f8f.png)

---
PR by @klonos: https://github.com/backdrop/backdrop/pull/1641

**Comment by klonos** [comment]
...while at it, I also changed https://github.com/backdrop/backdrop/blob/1.x/core/modules/path/path.path.inc#L39 from this:

`URL alias pattern for all @language @node_type URLs`

to this:

`URL alias pattern for all %node_type URLs in %language`

This makes it so that this sentence is grammatically correct when translated in Greek while still retaining the same meaning in English. @olafgrabienski and other international Backdrop people, please let me know if that string is easier/harder to translate in your native language.

**Comment by olafgrabienski** [comment]
@klonos Grammatically, your proposal is more close to the German language as well, but in my opinion the English sentence doesn't need to be changed. While the task of the English sentence is simply to express the thing in good English, the duty of a translation is to express it in another way which includes grammatical or other changes.

**Comment by klonos** [comment]
Thanx for taking the time to respond @olafgrabienski. Yes, fair enough. If there is consensus, I'll revert that change.

**Comment by serundeputy** [comment]
@klonos the issue was to get `allthethings` in `em`'s so the language change is a different issue.  I'd say in the interest of getting this committed lets keep the current language.  We can file a different issue for language change (probably better suited for the forum then the issue queue).

Otherwise we can wait on consensus on the language change up to you.  Do you want to revert the language or wait for consensus?

**Comment by rszrama** [comment]
Finally reviewed this ... meant to after the group call. In any event, patch looks great and +1 for keeping the language change. It makes sense to separate the language from the node type since they're both italicized - it better enables the user to distinguish between the language and the node type. Otherwise "_English Page_ URLs" could lead you to assume the node type is actually "English Page", not "Page".

**Comment by serundeputy** [comment]
@rszrama wanna give it a RTBC tag for posterity? I'll commit it tomorrow.

**Comment by serundeputy** [comment]
Thanks for the PR @klonos!
Thanks for the reviews @olafgrabienski and @rszrama 

This is merged into `1.x` and `1.5.x` and will ship with `1.5.3` and `1.6.0`!
